layers/best_practices/bp_wsi.cpp

```diff
@@ -277,7 +277,7 @@ void BestPractices::ManualPostCallRecordGetPhysicalDeviceSurfaceFormatsKHR(VkPhy
     if (auto bp_pd_data = Get<bp_state::PhysicalDevice>(physicalDevice)) {
         auto& call_state = bp_pd_data->vkGetPhysicalDeviceSurfaceFormatsKHRState;
 
-        if (*pSurfaceFormatCount) {
+        if (pSurfaceFormatCount) {
```

Comment:
```suggestion
        if (*pSurfaceFormatCount) {
```
The meaning is not the same. I think the original was checking for `*pSurfaceFormatCount > 0`, which here is equivalent to just checking `*pSurfaceFormatCount`. So I think we need to revert this change